Makefile

```diff
@@ -56,9 +90,9 @@ distclean: clean
 		perl merge.pl $< $(BASE_KCONF) | sort > $@ ; \
 	fi ;
 
-%.8: doc/%.8.txt doc/asciidoc.conf Makefile genkernel
-	a2x --conf-file=doc/asciidoc.conf --attribute="genkernelversion=$(PACKAGE_VERSION)" \
```

Comment:
i don't see the genkernel version being injected anywhere now; missing from this PR?

Reply:
I removed this due to issue of not having git repo when using dist pkg.

I can add a code that generates a file with version upon `make dist`. The manpage generation code can then check for this file as well as the git to determine the version to inject. 

Any preference on the file path?

Reply:
addressed in commit [c295e6b](https://github.com/gentoo/genkernel/pull/56/commits/c295e6b7ce4efa23d5b797e24c245e360a85c03b

lemme know if you think it needs to be done differently